src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -572,6 +606,14 @@ namespace winrt::Microsoft::Terminal::Control::implementation
 
         _settings = settings;
 
+        // GH#11285 - If the user is on Windows 10, and they wanted opacity, but
+        // didn't explicitly request acrylic, then opt them in to acrylic.
+        // On Windows 11+, this isn't needed, because we can have vintage opacity.
+        if (!_isVintageOpacityAvailable() && _settings.Opacity() < 1.0 && !_settings.UseAcrylic())
+        {
+            _settings.UseAcrylic(true);
+        }

```

Comment:
This is fine of course, but I don't understand why this code is redundant with the one in `AdjustOpacity`. Why isn't it enough to only do this in one of two places?

Reply:
One of them does it when the control is first created, the other place does it when the control adjusts opacity via the scrollwheel.